conjurapi/config_test.go

```diff
@@ -26,75 +27,70 @@ func TempFileForTesting(prefix string, fileContents string) (string, error) {
 }
 
 func TestConfig_IsValid(t *testing.T) {
-	Convey("Return without error for valid configuration", t, func() {
+	t.Run("Return without error for valid configuration", func(t *testing.T) {
 		config := Config{
 			Account:      "account",
 			ApplianceURL: "appliance-url",
 		}
 
 		err := config.validate()
-
-		So(err, ShouldBeNil)
+		assert.NoError(t, err)
 	})
 
-	Convey("Return error for invalid configuration", t, func() {
+	t.Run("Return error for invalid configuration", func(t *testing.T) {
 		config := Config{
 			Account: "account",
 		}
 
 		err := config.validate()
```

Comment:
There are two test cases that call `config.validate()` that could be combined into a table of test cases.